src/assets/custom/js/custom.js

```diff
@@ -74,43 +74,20 @@ $(window).resize(function () {
     equalizeHeights();
 });
 
-
-(function navbarActive(){
-	$(function() {
-		const url = window.location.href;
-		$(".nav a").each(function() {
-			if (url.includes(this.href)) {
-				$(this).closest("li").addClass("active");
-			}
-		});
-	});
-})();
-
-(function navbarProjectsActive(){//TODO delete once we add projects to the navbar
-	$(function() {
-		const url = window.location.href;
-		$(".nav a").each(function() {
-			if (url.includes("/projects") && this.href.includes("/client")) {
-				$(this).closest("li").addClass("active");
-			}
-		});
-	});
-})();
```

Comment:
Are these needed if the feature-toggle is `off`?

Incidentally when is the feature-toggle going to be removed?

Reply:
Good point. 

Since this ticket completes the epic, I will also remove the feature toggle's as part of this PR.